lib/Config.php

```diff
@@ -162,7 +183,7 @@ public function get_connection($name)
 	 *
 	 * @return string
 	 */
-	public function get_default_connection_string()
+	public function get_default_connection_info()
```

Comment:
Not exactly no, maybe we're better of tagging the next release 2.0.0.. But I think that's @jpfuentes2 call.